src/Modal/index.jsx

```diff
@@ -50,6 +50,8 @@ class Modal extends React.Component {
 
   componentWillUnmount() {
     if (this.parentElement) {
+      // TODO: update this to use the new createRoot() compatible APIs.
```

Comment:
If this `TODO` is one that's going to stick around after this PR lands it'd probably be good to make a GH issue for it

Reply:
Created issue: https://github.com/openedx/paragon/issues/3382